From 0012c8b6583c84dbdb4f8e81a1f4c27531010e28 Mon Sep 17 00:00:00 2001
From: Victor Gu <xigu@marvell.com>
Date: Tue, 23 Aug 2016 17:26:39 +0800
Subject: [PATCH 0706/2241] arm64: configs: enable PCIe driver for Armada 3700

This patch enables the driver for the PCIe host controller found on
the Marvell Armada 3700 ARM64 SoCs.

Change-Id: I2d0f7ab52f82979f52f33950339fb2e3bd5c806d
Signed-off-by: Victor Gu <xigu@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/32128
Reviewed-by: Wilson Ding <dingwei@marvell.com>
Tested-by: Wilson Ding <dingwei@marvell.com>
---
 arch/arm64/configs/mvebu_v8_lsp_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/mvebu_v8_lsp_defconfig b/arch/arm64/configs/mvebu_v8_lsp_defconfig
index 44d2787..80ff706 100644
--- a/arch/arm64/configs/mvebu_v8_lsp_defconfig
+++ b/arch/arm64/configs/mvebu_v8_lsp_defconfig
@@ -38,6 +38,7 @@ CONFIG_IGNORE_ASYNC_ABORT=y
 CONFIG_PCI_HOST_GENERIC=y
 CONFIG_PCIE_IPROC=y
 CONFIG_PCIE_ARMADA_8K=y
+CONFIG_PCI_AARDVARK=y
 # CONFIG_MMIO_64BIT is not set
 CONFIG_SCHED_MC=y
 CONFIG_PREEMPT=y
-- 
2.7.4

